composer.json

```diff
@@ -13,6 +13,8 @@
     ],
     "require": {
         "php": "^7.2.9",
+        "symfony/config": "^4.4|^5.0",
+        "symfony/dependency-injection": "^4.4|^5.0",
```

Comment:
Why did you add those ? Their are useless :)

You must add **symfony/http-client** like google-mailer please : https://github.com/symfony/google-mailer/blob/master/composer.json

Reply:
Because when you want to add feature in your package, my IDE throw missing package error in folder `DependecyInjection`.
They are not useless here!

Reply:
BTW https://github.com/symfony/google-mailer don't have this dependency because its doesnt use DepencyInjection

Reply:
Wich IDE are you using and where is your **DependecyInjection** error ?
Mine doesn't say anything because they are already specified if you follow this package dependencies :
**symfony/mailer -> symfony/mime -> symfony/dependency-injection -> symfony/config**

Then if you look in any ***-mailer** bridge's composer.json of Symfony repositories, there are never mentionned.
Because for me, this bridge is just a part of symfony/mailer and it's to this package to get the responsibility of require them.